### PR TITLE
recipes-qt: Add a sample QtWPE mini browser application

### PR DESCRIPTION
**Pull request by philn**

### CODE REVIEW

**Comment by psaavedra** [comment]
This change doesn't affects to the test build preset: https://gitlab.com/saavedra.pablo/meta-perf-browser/pipelines/47476825

I wonder if I should to create a additional one to build a  QT image.

**Comment by philn** [comment]
Sounds like a good idea :)
- it will work only with a git snaphot or the upcoming 2.24 wpewebkit release
- enable qtwpe in wpewebkit's config
- in your image add these packages:
```
IMAGE_INSTALL_append = " wpewebkit-qtwpe-qml-plugin qt-wpe-simple-browser"

IMAGE_INSTALL_append = " qtbase-dev qtbase-mkspecs qtbase-plugins qtbase-staticdev qttranslations-qtbase \
    qtdeclarative-dev qtdeclarative-mkspecs qtdeclarative-qmlplugins qtdeclarative-tools qtdeclarative-staticdev \
    qtimageformats-dev qtimageformats-plugins \
    qtscript-dev qtscript-mkspecs \
    qtsvg-dev qtsvg-mkspecs qtsvg-plugins \
    qttools-dev qttools-mkspecs qttools-staticdev qttools-tools \
    qtwayland-dev qtwayland-mkspecs qtwayland-plugins qtwayland-tools \
    qtquickcontrols2 qtquickcontrols2-dev qtquickcontrols2-mkspecs"
```

**Comment by psaavedra** [comment]
Added in the pipeline for trunk: https://gitlab.com/saavedra.pablo/meta-perf-browser/commit/e18e11182101640b3c5b0312d73d3783b83d4a51

First build: https://gitlab.com/saavedra.pablo/meta-perf-browser/-/jobs/166887931

File: recipes-qt/qt-wpe-simple-browser/qt-wpe-simple-browser_0.1.bb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+SUMMARY = "Qt5 WPE simple browser application"
+LICENSE = "LGPL-2.1+"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/COPYING;md5=751419260aa954499f7abaabaa882bbe"
```
[the remark this answers is not in the document]
Comment (philn) [reply recipes-qt/qt-wpe-simple-browser/qt-wpe-simple-browser_0.1.bb]: Nice. I could have updated the PR before merge, there was no rush to commit this. Will submit a follow-up PR then...

**Comment by psaavedra** [comment]
> Added in the pipeline for trunk: https://gitlab.com/saavedra.pablo/meta-perf-browser/commit/e18e11182101640b3c5b0312d73d3783b83d4a51
> 
> First build: https://gitlab.com/saavedra.pablo/meta-perf-browser/-/jobs/166887931

Success: https://gitlab.com/saavedra.pablo/meta-perf-browser/-/jobs/166952447

File: recipes-qt/qt-wpe-simple-browser/qt-wpe-simple-browser_0.1.bb (hunk 1 above)
[the remark this answers is not in the document]
Comment (psaavedra) [reply recipes-qt/qt-wpe-simple-browser/qt-wpe-simple-browser_0.1.bb]: Don't worry, it wasn't a mandatory change and I started to prefer to test the QT build as well.

**Pull request merged**